catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/query/cql/CqlRequest.java

```diff
@@ -203,6 +223,22 @@ private String parseSrc(String localSource) {
     return src;
   }
 
+  private void parseSrcs(String localSource) {
+    for (int i = 0; i < srcs.length; i++) {
+      if (StringUtils.equalsIgnoreCase(srcs[i], LOCAL_SOURCE) || StringUtils.isBlank(srcs[i])) {
+        srcs[i] = localSource;
+      }
+    }
+  }
+
+  public String getSourceResponseString() {
+    if (srcs != null && srcs.length != 0) {
+      return Arrays.toString(srcs);
```

Comment:
❗️ `Arrays.toString` string representation is not defined.  

https://docs.oracle.com/javase/8/docs/api/java/util/Arrays.html#toString-java.lang.Object:A-

Use a collection and `String.join` to control the format.

https://docs.oracle.com/javase/8/docs/api/java/lang/String.html#join-java.lang.CharSequence-java.lang.Iterable-